src/Compilers/VisualBasic/Portable/Binding/BinderFactory.vb

```diff
@@ -387,8 +387,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                  SyntaxKind.GetAccessorStatement,
                                  SyntaxKind.AddHandlerAccessorStatement,
                                  SyntaxKind.RemoveHandlerAccessorStatement,
-                                 SyntaxKind.RaiseEventAccessorStatement
-                                ' Default values are not valid (and not bound) for lambda parameters or property accessors
+                                 SyntaxKind.RaiseEventAccessorStatement,
+                                 SyntaxKind.OperatorStatement
+                                ' Default values are not valid (and not bound) for these types of statements
```

Comment:
I'm nto sure how i feel about that.  A primary purpose of hte Roslyn APIs is to give us semantic information, even for invalid code.